src/webview_provider/docsEditPanel.ts

```
@@ -545,6 +549,25 @@ export class DocsEditViewPanel implements WebviewViewProvider {
 
         const { command, syncRequestId, ...params } = message;
         switch (command) {
+          case "generateTestsForColumns":
+            const testSuggestionsForModel =
+              (await this.dbtTestService.generateTestsForColumns(
+                project,
+                this._panel,
+              )) || ({} as Model);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)